corehq/apps/ota/views.py

```diff
@@ -361,4 +361,9 @@ def form_valid(self):
 @login_or_digest_or_basic_or_apikey()
 @require_GET
 def heartbeat(request, domain, id):
-    return JsonResponse({})
+    # mobile needs this. This needs to be revisited to actually work dynamically (Sravan June 7, 17)
+    return JsonResponse({
+        "app_id": "value_copied_from_app_id_url_param",
```

Comment:
I have fixed this. Sorry about it, just to confirm per the spec the `app_id` should come from the last section of URL https://www.commcarehq.org/a/{domain}/phone/heartbeat/{app_id}/. it's not a get URL param, and that's what I changed this to